### PR TITLE
pferd 3.4.2 (new formula)

### PR DESCRIPTION
**Pull request by pavelzw**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/pferd.rb [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    url "https://files.pythonhosted.org/packages/d6/04/255c68974ec47fa754564c4abba8f61f9ed68b869bbbb854198d6259c4f7/yarl-1.8.1.tar.gz"
+    sha256 "af887845b8c2e060eb5605ff72b6f2dd2aab7a761379373fd89d314f4752abbf"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    system "#{bin}/pferd", "--version"
```
Comment (chenrui333) [inline Formula/pferd.rb:107]: assert the response for this?
```suggestion
    system "#{bin}/pferd", "--version"
```

File: Formula/pferd.rb [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    system "#{bin}/pferd", "--help"
```
Comment (chenrui333) [inline Formula/pferd.rb:108]: probably dont need this
```suggestion
```

File: Formula/pferd.rb [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+    assert_match "Error Failed to load config", shell_output("#{bin}/pferd", 1)
+
+    (testpath/"pferd.cfg").write <<~EOS
+      [crawl:Foo]
+      type = kit-ilias-web
+      target = 1234567
+    EOS
+    assert_match "key 'auth': Missing value", shell_output("#{bin}/pferd -c #{testpath}/pferd.cfg", 1)
```
Comment (chenrui333) [inline Formula/pferd.rb:117]: 👍

**Review by chenrui333 — COMMENTED** [review]
Looks good!

**Review by chenrui333 — APPROVED** [review]

File: Formula/pferd.rb [hunk 4]
```diff
@@ -0,0 +1,118 @@
+class Pferd < Formula
+  include Language::Python::Virtualenv
+
+  desc "Programm zum Flotten Einfachen Runterladen von Dateien"
```
Comment (chenrui333) [inline Formula/pferd.rb:4]: submit an issue for this part, https://github.com/Garmelon/PFERD/issues/64

**Review by SMillerDev — APPROVED** [review]

**Comment by SMillerDev** [comment]
Thanks @pavelzw  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**